app/views/pages/_sponsor_image.html.erb

```diff
@@ -1,2 +0,0 @@
-<img src="<%= optimized_image_url(sponsor.nav_image_url || sponsor.dark_nav_image_url || sponsor.profile_image_url) %>" width="250" alt="<%= sponsor.name %> logo" loading="lazy" class="partner-image-light-mode">
```

Comment:
Deleted as it was referenced in app/views/articles/_single_sponsor.html.erb which was deleted.